site/templates/message.html

```diff
@@ -53,122 +56,134 @@ <h2>Send Brian a message on his Tidbyt.</h2>
             <option value="CG-pixel-3x5-mono">CG-pixel-3x5-mono</option>
             <option value="CG-pixel-4x5-mono">CG-pixel-4x5-mono</option>
           </select>
-          <div class="form-text"><a href="https://github.com/tidbyt/pixlet/blob/main/docs/fonts.md">Font Examples</a>
+          <div class="form-text">
+            <a href="https://github.com/tidbyt/pixlet/blob/main/docs/fonts.md">Font Examples</a>
           </div>
         </div>
 
         <div class="row justify-content-center">
           <div class="col-auto">
             <button id="submit-button" type="submit" class="btn btn-primary">
-              <span id="submit-button-span" style="display: none" class="spinner-grow spinner-grow-sm" role="status"
-                aria-hidden="true"></span>
-              Send Message
+              <span class="spinner-grow spinner-grow-sm" role="status" aria-hidden="true"></span>
+              <span class="text">Send Message</span>
```

Comment:
Initially you were grabbing the `innerHTML` of the whole button and running a `replace` to set the text.

That worked, but you can also surround text with an arbitrary `span` or `div` and a class/ID name, and then target that specifically.